docs/wmco-prerequisites.md

```diff
@@ -29,7 +29,9 @@ these errors, only use the appropriate version according to the cloud provider i
 The following tables outline the type of networking configuration and Windows Server versions to be used based on your 
 cloud provider. The network configuration must be completed during the installation of the cluster.
   
-Note: OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+Note: 
+* OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+* Multiple networks (dual NICs) is NOT supported by WMCO.
```

Comment:
This is not related to multiple networks. This is about dual NIC. There are ways to connect same NIC with two different subnets ( IPs).  Let's just make a note saying 
Dual NIC is not supported by WMCO.

Reply:
@jrvaldes @selansen Do we know if dual NICs are supported by OpenShift? I don't see that phrase in our docs. If we go with _Dual NIC is not supported by WMCO._, folks could assume that OCP does support dual NIC.

Reply:
@mburke5678   I tend to agree with you. That's why I went with `Multiple networks` instead of `Dual NIC`.

Reply:
Updated to `Dual NIC is not supported by WMCO` as per @selansen  request.

Reply:
@mburke5678 I read in OpenShift docs and  learnt they do support dual (Multiple ) NIC on the Linux side.
Pls check this one.
https://www.redhat.com/en/blog/multus-takes-leading-role-container-networking